ffs/types.go

```diff
@@ -57,7 +57,7 @@ type JobStatus int
 
 const (
 	// Queued indicates the Job is queued in the Scheduler.
-	Queued JobStatus = iota
+	Queued JobStatus = iota + 1
```

Comment:
Thought that starting enums at 1 would be safer. Just to avoid that the default value is `Queued`, and some client (or some internal code) that fogets checking errors interprets a "valid" status which shouldn't be considered.

This idea might apply to many places (and repos!)... so kind of throwing the idea to see feedback.

Reply:
Seems like a pretty good idea. Protobuf style recommends always to have the 0 value as the "unspecified" value:

> The zero value enum should have the suffix UNSPECIFIED

Maybe that could work well in Go as well.

Reply:
That's from https://developers.google.com/protocol-buffers/docs/style#enums

Reply:
Looks like we need to update the rpc layer to be safe mapping to the proto version of the enum... we're just assuming the int values are equal, we need to check explicitly. PR coming up